mypy_django_plugin/transformers/models.py

```diff
@@ -88,6 +88,7 @@ def create_new_var(self, name: str, typ: MypyType) -> Var:
         var._fullname = self.model_classdef.info.fullname + "." + name
         var.is_initialized_in_class = True
         var.is_inferred = True
+        var.has_explicit_value = True
```

Comment:
See #2803, for this. I'm not sure how to do stacked pr from forks, I'll rebase to clear this once it lands